README.md

```diff
@@ -0,0 +1,171 @@
+Vimerator-labs
+==============
+
+![amo release][amo_version] ![github release][github_version]
```

Comment:
Would be cool if these also linked to the AMO and GitHub release pages, like so:

    [![amo release][amo_version]][amo_release] [![github release][github_version]][github_release]

    [amo_version]: https://img.shields.io/amo/v/vimperator.svg
    [amo_release]: https://addons.mozilla.org/en-US/firefox/addon/vimperator/
    [github_version]: https://img.shields.io/github/release/vimperator/vimperator-labs.svg
    [github_release]: https://github.com/vimperator/vimperator-labs/releases/latest